static/src/js/util/metaIcon.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react'
```

Comment:
We should add a test for this component.

Reply:
This isn't a new component, it's just relocated code from CollectionResultsItem.js to its own file in 6d42317 since it is now also being called from AccessMethodRadio.js. It's still being tested through the those other two scripts.